src/browser/api/window.js

```diff
@@ -1251,6 +1253,21 @@ Window.hide = function(identity) {
     browserWindow.hide();
 };
 
+Window.isNotification = function(name) {
+    const noteGuidRegex = /^A21B62E0-16B1-4B10-8BE3-BBB6B489D862/;
+    return noteGuidRegex.test(name);
+};
+
+Window.isNotificationType = function(identity, callback = () => {}) {
+    const { name } = identity;
+
+    const isNotification = Window.isNotification(name);
+    const isQueueCounter = name === Window.QUEUE_COUNTER_NAME;
```

Comment:
This step would not be necessary if https://appoji.jira.com/browse/RUN-2818 were taken care of.